modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -3082,12 +3082,34 @@ definition {
 	}
 
 	@description = "Verify that a message warning is displayed when the user adds another transition and selects it as default"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test WarningIsDisplayedWhenNewTransitionDefault {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#WarningIsDisplayedWhenNewTransitionDefault";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "20,-700",
```

Comment:
The `nodePosition` now needs to be (0,0) to put the task node on a position that have not impact with other nodes, or do not pass the `nodePosition` at all, thanks!